### PR TITLE
Update validation constants

### PR DESCRIPTION
**Pull request by JustinPeng13**

### CODE REVIEW

File: src/main/java/seedu/guest/model/guest/Email.java [hunk 1]
```diff
@@ -28,14 +28,14 @@ public class Email {
             + "and adhere to the following constraints:\n"
             + "1. The local-part should only contain alphanumeric characters and these special characters, excluding "
             + "the parentheses, (" + SPECIAL_CHARACTERS + "). The local-part may not start or end with any special "
-            + "characters and can be at most 64 characters long.\n"
+            + "characters and can be at most " + MAX_LOCAL_LENGTH + " characters long.\n"
             + "2. This is followed by a '@' and then a domain name. The domain name is made up of domain labels "
             + "separated by periods.\n"
             + "The domain name must:\n"
             + "    - end with a domain label at least 2 characters long\n"
```
Comment (jehousoh) [inline src/main/java/seedu/guest/model/guest/Email.java]: Should we have a MIN_DOMAIN_LENGTH as well?

**Review by jehousoh — APPROVED** [review]
lgtm

File: src/main/java/seedu/guest/model/guest/NumberOfGuests.java [hunk 2]
```diff
@@ -9,9 +9,14 @@
...
 public class NumberOfGuests {
 
+    // value constraints
+    private static final int MIN_NUM_GUESTS = 1;
+    private static final int MAX_NUM_GUESTS = 4;
+
     public static final String MESSAGE_CONSTRAINTS =
-            "Number of Guests should only contain 1 numerical character from 1 to 4.";
-    public static final String VALIDATION_REGEX = "\\d{1,}";
+            "Number of Guests should only contain 1 numeric character from " + MIN_NUM_GUESTS + " to "
```
Comment (bryanhce) [inline src/main/java/seedu/guest/model/guest/NumberOfGuests.java]: Should this be NumberOfGuests?

**Review by bryanhce — COMMENTED** [review]
1 fix?

File: src/main/java/seedu/guest/model/guest/NumberOfGuests.java (hunk 2 above)
Comment (JustinPeng13) [reply src/main/java/seedu/guest/model/guest/NumberOfGuests.java]: Changed!

**Pull request merged**